ios/FluentUI/Avatar/AvatarTokenSet.swift

```diff
@@ -109,7 +109,7 @@ public class AvatarTokenSet: ControlTokenSet<AvatarTokenSet.Tokens> {
                 return .float({
                     switch size() {
                     case .size16, .size20, .size24:
-                        return GlobalTokens.borderSize(.thin)
+                        return GlobalTokens.borderSize(.thinner)
```

Comment:
Shouldn't the border size remain unchanged or did design specify this?

Reply:
Looks like the size stays the same; the name is changing.

| Before | After |
|---|---|
| `.thin == 1.0` | `.thinner == 1.0`, `.thin == 1.5` |

So changing from `.thin` to `.thinner` is how we keep the size the same.